plugins/events-backend/README.md

```diff
@@ -38,25 +52,20 @@ Additionally, add the events plugin to your backend.
 // [...]
 +  const eventsEnv = useHotMemoize(module, () => createEnv('events'));
 // [...]
-+  apiRouter.use('/events', await events(eventsEnv, []));
++  apiRouter.use('/events', await events(eventsEnv));
 // [...]
 ```
 
 ### With Event-based Entity Providers
 
 In case you use event-based `EntityProviders`,
-you may need something like the following:
+you will need to add the subscription to the catalog plugin creation:
 
 ```diff
-// packages/backend/src/index.ts
--  apiRouter.use('/events', await events(eventsEnv, []));
-+  apiRouter.use('/events', await events(eventsEnv, eventBasedEntityProviders));
+// packages/backend/src/plugins/catalog.ts
++
```

Comment:
The diff is empty now, there is no change mentioned for the catalog.ts file.

Also, it is not clear what is meant by

> you will need to add the subscription to the catalog plugin creation:

I guess you miss something like

```env.eventBroker.subscribe(entityProviderWithEventSupport);```